site/source/_posts/2019/01/28/Version-5-Dojo.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Dojo 5
+date: 2019-01-28 08:00:00
+authorId: unknown
+featured_image: featured.jpg
+---
+
+Dojo version 5 has arrived!
+
+We're excited to announce the 5.0.0 release of Dojo. This version builds on previous releases with a vast array of bug fixes and improvements throughout the framework.
+
+{% asset_img featured.jpg feature-full %}
+
+### Conditional Polyfills - Serving Less JavaScript
+
+One of our primary goals for modern Dojo is to improve performance by serving minimal JavaScript bundles by default. For version 5, we're happy to announce an out of the box solution to optimize the user experience for dealing with bundling and loading polyfills in Dojo applications.
+
+By using `@dojo/framework/shim` ponyfills for [`WebAnimations`][WebAnimationsAPI], [`IntersectionObserver`][IntersectionObserverAPI], [`ResizeObserver`][ResizeObserverAPI], and [`fetch`][FetchAPI] the Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
```

Comment:
I'd like to remove these explicitly listed polyfills from the "serving less javascript" section. as it's a build pattern and not just relevant to these ponyfills. 

We should write a new section saying we've now added additional third party polyfills to shim, and they can/should be imported like any other ponyfill in shim. And mention the user no longer has to explicitly import the third party script or polyfill.

Reply:
We should also briefly describe each of the polyfills with the appropriate link to WICG or whatever